src/pudl/transform/eia923.py

```diff
@@ -636,6 +636,7 @@ def generation_fuel(eia923_dfs, eia923_transformed_dfs):
         "total_fuel_consumption_mmbtu",
         "elec_fuel_consumption_mmbtu",
         "net_generation_megawatthours",
+        "early_release",
```

Comment:
I've added these new `early_release` columns in here as columns to drop *for now*. I'd like to put this aside until we get somewhere with #1799 (already have an implement issue #1796